react-frontend/src/components/products/productscard.js

```diff
@@ -0,0 +1,18 @@
+import React from 'react';
+import 'antd/dist/antd.css';
+import { Card } from 'antd';
+import '../../css/card.css';
+
+function ProductsCard(props) {
+  let cardHeight = 0;
```

Comment:
i feel theres a better way of designing this component. If this is a flexible component we should provide a framework of how to use it. 

For example you can provide two implementations to this component 

```
const ProductsCard = ({render}) => (<Card> {render()}</Card>);

<ProductsCard render={() => <div>....</div>} />